Gems/ScriptCanvasPhysics/Code/Source/WorldNodes.h

```diff
@@ -120,6 +121,43 @@ namespace ScriptCanvasPhysics
             "EntityId",
             "Surface");
 
+        AZ_INLINE Result RayCastFromScreenWithGroup(
+            const AZ::Vector2& screenPosition,
+            float distance,
+            const AZStd::string& collisionGroup,
+            AZ::EntityId ignore)
+        {
+            AZ::EntityId camera;
+            Camera::CameraSystemRequestBus::BroadcastResult(camera, &Camera::CameraSystemRequestBus::Events::GetActiveCamera);
+            if (camera.IsValid())
+            {
+                AZ::Vector3 origin = AZ::Vector3::CreateZero();
+                Camera::CameraRequestBus::EventResult(origin, camera, &Camera::CameraRequestBus::Events::ScreenToWorld, screenPosition, 0.f);
+                AZ::Vector3 offset = AZ::Vector3::CreateZero();
+                Camera::CameraRequestBus::EventResult(offset, camera, &Camera::CameraRequestBus::Events::ScreenToWorld, screenPosition, 1.f);
+                const AZ::Vector3 direction = (offset - origin).GetNormalized();
+                return RayCastWorldSpaceWithGroup(origin, direction, distance, collisionGroup, ignore);
+            }
+
+            // fallback in the rare case there is no active camera
+            return AZStd::make_tuple(false, AZ::Vector3::CreateZero(), AZ::Vector3::CreateZero(), 0.0f, AZ::EntityId(), AZ::Crc32());
+        }
+
+        SCRIPT_CANVAS_GENERIC_FUNCTION_NODE(RayCastFromScreenWithGroup,
```

Comment:
This type of node is on the deprecation path. Unless you don't want them in Lua, you can just expose these to behavior context.

Reply:
Good to know - what is the correct replacement method?  
If I make a PhysX Raycast BehaviorContext method that returns an tuple, will that show up in ScriptCanvas as multiple outputs?  And will it work OK in Lua?